4-float-features/4-float-features.md

```diff
@@ -0,0 +1,74 @@
+# Introduction
+
+Num type has been separated into Int type and Float type.\
```

Comment:
use backticks around code constructs like `Int` and `Float` to typeset them properly